modules/rank-eval/src/test/java/org/elasticsearch/index/rankeval/ReciprocalRankTests.java

```diff
@@ -146,4 +148,44 @@ public void testXContentRoundtrip() throws IOException {
         }
         return hits;
     }
+    
+    public void testSerialization() throws IOException {
+        ReciprocalRank original = new ReciprocalRank();
+        original.setRelevantRatingThreshhold(randomIntBetween(0, 20));
```

Comment:
Might make sense to also create a `createTestItem` method here, even though currently there's not much to do, but this way we don't miss any test method when we might add more parameters to the metric later.